python/ray/tune/trial.py

```diff
@@ -83,9 +88,11 @@ def __init__(
         in ray.tune.config_parser.
         """
 
-        if not ray.tune.registry._default_registry.contains(
-                ray.tune.registry.TRAINABLE_CLASS, trainable_name):
-            raise TuneError("Unknown trainable: " + trainable_name)
+        if not has_trainable(trainable_name):
+            # Make sure rllib agents are registered
+            from ray import rllib  # noqa # pylint: disable=unused-import
```

Comment:
Nite, the comment should just be `# noqa: F401`

Reply:
Changed